FinSearchUnified/Subscriber/Frontend.php

```
@@ -150,20 +172,19 @@ public function onFrontendPostDispatch(Enlight_Event_EventArgs $args)
         $searchResultContainer = Shopware()->Config()->get('SearchResultContainer');
         $navigationContainer = Shopware()->Config()->get('NavigationContainer');
 
-        try {
```

This try/catch should stay too.

---

There was no Enlight_Exception being thrown in the code. Should I change it to a Exception instead?

---

Yes, please. In any error case this shouldn't break the shop but fail silently.